### ISSUE TITLE
[Bug]: Scaling a Textbox object in a Group selection does not adjust text to multiline

### ISSUE THREAD

**Issue by mirasnag** [fabricjs/fabric.js#10326]
### CheckList

- [X] I agree to follow this project's Code of Conduct
- [X] I have read and followed the [Contributing Guide](https://github.com/fabricjs/fabric.js/blob/master/CONTRIBUTING.md)
- [X] I have read and followed the [Issue Tracker Guide](https://github.com/fabricjs/fabric.js/blob/master/CONTRIBUTING.md#%EF%B8%8F-issue-tracker)
- [X] I have searched and referenced existing issues and discussions
- [X] I am filing a **BUG** report.
- [X] I have managed to reproduce the bug **after** upgrading to the **latest** version
- [X] I have created an accurate and minimal reproduction

### Version

6.0.2

### In What environments are you experiencing the problem?

Chrome

### Node Version (if applicable)

None

### Link To Reproduction

https://fabricjs.com/demos/super-sub-script/

### Steps To Reproduce

1. Go to this official demo: https://fabricjs.com/demos/super-sub-script/
2. Select both the Textbox and IText objects
3. Scale the width of the group

### Expected Behavior

When scaling the group, the Textbox object should retain its text size and wrap the text into multiple lines, similar to how it behaves when scaled individually.
![Screen Shot 2024-12-09 at 11 21 04](https://github.com/user-attachments/assets/69be54e4-2de9-4377-a90f-e3c76ef78972)
![Screen Shot 2024-12-09 at 11 21 15](https://github.com/user-attachments/assets/b5686e10-38c6-4f4f-9120-c0a18ed59e00)

### Actual Behavior

When scaled within a group, the Textbox is scaled proportionally, similar to the IText object, without wrapping its text into multiple lines.
![Screen Shot 2024-12-09 at 11 21 32](https://github.com/user-attachments/assets/e8175fca-701b-4d6c-93f9-a384f8eaf69d)
![Screen Shot 2024-12-09 at 11 21 44](https://github.com/user-attachments/assets/8791a032-6b88-4da6-b02e-7c8a506aad9d)

**Comment by asturur** [comment]
Yes this because the active selection, the temporary group that is handling all the objects scales itself and not the objects inside. What you can do is attach a scaling event to the active selection and correct the textbox size step by step but is going to be complicated.

Is not a bug is how active selection works. If you don't like that the simplest solution is to disable the ml, mt, mr and mb controls of the active selection and do not allow for non uniform scaling when multi selected.
